pandas/core/computation/common.py

```diff
@@ -1,9 +1,12 @@
 import numpy as np
 
-from pandas.compat import reduce
+from pandas.compat import reduce, string_types
 
 import pandas as pd
 
+# A token value Python's tokenizer probably will never use.
```

Comment:
can you instead use use ``str(uuid())`` appended to the columm name?

Reply:
Yes.